client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -1938,6 +1938,17 @@ Undocumented.prototype.unregisterDevice = function( deviceId, fn ) {
 };
 
 /**
+ * Requests streamlined approval to WordAds program
+ *
+ * @param {int}       siteId            The site ID
+ * @returns {XMLHttpRequest}          The XHR instance
+ */
+Undocumented.prototype.wordAdsApprove = function( siteId ) {
```

Comment:
Thanks @retrofox ! For the test, lets leave it in undocumented and if we decide to have it in wpcom, then I will ask you to move it :)